src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -2418,6 +2418,73 @@ def test_vmss_update(self, resource_group):
         # test that cannot try to update protection policy on VMSS itself
         self.cmd('vmss update -g {rg} -n {vmss} --protect-from-scale-in True --protect-from-scale-set-actions True', expect_failure=True)
 
+    @live_only()
+    @ResourceGroupPreparer(name_prefix='cli_test_vmss_update_', location='westus2')
+    def test_vmss_update_cross_tenant(self, resource_group):
+        self.kwargs.update({
+            'location': 'westus2',
+            'another_rg': self.create_random_name('cli_test_vmss_update_', 40),
+            'vm': self.create_random_name('cli_test_vmss_update_', 40),
+            'image_name': self.create_random_name('cli_test_vmss_update_', 40),
+            'aux_sub': 'bead59b7-f469-4601-803a-790729c5213d',
```

Comment:
Can it pass using `azureclitest@azuresdkteam.onmicrosoft.com`?

Reply:
No, `azureclitest` has no other subs permissions.

Reply:
Permission added. `azureclitest` can pass now.